lms/templates/seq_module.html

```diff
@@ -13,6 +13,16 @@
         </div>
       </div>
     % endif
+    % if gated_banner:
+      <div class="pattern-library-shim alert alert-information subsection-header" tabindex="-1">
+        <span class="pattern-library-shim icon alert-icon icon-bullhorn" aria-hidden="true"></span>
+        <div class="pattern-library-shim alert-message">
+          <p class="pattern-library-shim alert-copy">
+            ${_("This subsection is unlocked for learners when they meet the pre-requisite requirements.")}
```

Comment:
@kevinjkim Only one minor tweak needed (remove the hyphen): "pre-requisite" -> "prerequisite" 
👍 from me with that change. Thanks!